app/src/main/java/ca/bc/gov/bchealth/ui/healthrecord/add/AddHealthRecordsFragment.kt

```diff
@@ -33,59 +33,54 @@ class AddHealthRecordsFragment : Fragment(R.layout.fragment_health_records) {
     private val viewModel: AddHealthRecordsOptionsViewModel by viewModels()
     private val bcscAuthViewModel: BcscAuthViewModel by viewModels()
     private val sharedViewModel: SharedViewModel by activityViewModels()
-
-    override fun onCreate(savedInstanceState: Bundle?) {
-        super.onCreate(savedInstanceState)
-    }
+    private var destinationId = 0
```

Comment:
keeping this in sharedviewModel will preserve if your fragment gets re-created.

Reply:
yes, this is implemented.